lib/cli/src/commands/run.rs

```diff
@@ -58,8 +57,6 @@ static WASMER_HOME: Lazy<PathBuf> = Lazy::new(|| {
 /// The unstable `wasmer run` subcommand.
 #[derive(Debug, Parser)]
 pub struct Run {
-    #[clap(flatten)]
-    verbosity: clap_verbosity_flag::Verbosity<WarnLevel>,
```

Comment:
We should keep the verbosity flag IMO.

Lots of users won't know about RUST_LOG.

Reply:
Agreed

Reply:
@theduke and @syrusakbary what are the semantics you want for those verbosity flags? Should `-vvv` increase the default tracing log level from WARN to TRACE, or should it be affecting how much human-friendly output we are showing (progress bars, etc.)?

Also, what do we want the `--quiet` flag to do? Using `clap_verbosity_flag`, adding one `-q` will decrease the default tracing log level from WARN to ERROR.